src/engraving/libmscore/arpeggio.cpp

```diff
@@ -97,49 +90,43 @@ void Arpeggio::draw(mu::draw::Painter* painter) const
     case ArpeggioType::NORMAL:
     case ArpeggioType::UP:
     {
-        RectF r(symBbox(m_symbols));
+        const RectF& r = m_layoutData.symsBBox;
```

Comment:
I'm afraid this will break the `stable` version of layout, because that doesn't calculate `m_layoutData.symsBBox`. The question is: should we care about that? If we answer "yes", then it will limit development speed of course; if we answer "no", then it will limit the usefulness of having a `dev` and `stable` version.

Reply:
Thank you, I noticed that I broke the "stable" version in BagpipeEmbellishment, but I didn't think about it.
These are very interesting questions...
I see solutions like this:
1. Move the draw to the layout subsystems (it is possible to call it differently), i.e. consider that the layout and draw are inseparable from each other, and accordingly they will be synchronous with each other and a simple switch will remain. Including for palettes there will be a "own" draw.
2. Now take care of backward compatibility, and hope that it will be easier later, hope that later the layouts will fill the same structure LayoutData, only in their own way.

Reply:
Fixed for paletes and stable layout